CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.5.1
+* FIX. Filtering doesn't work when Hierarchy mode of showing date field was chosen
+
```

Comment:
Please do not remove `package-lock.json`.
It allows to load the same dependencies each time we build the visual.